tests/python/contrib/test_dnnl.py

```diff
@@ -249,24 +492,93 @@ def get_graph():
     run_and_verify_func(get_graph(), run_module=run_module, dtype=dtype)
 
 
-def test_unary(run_module):
+def test_elementwise(run_module, dtype="float32"):

```

Comment:
should we add markers to indicate that LLVM and DNNL are required for those tests?